src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java

```diff
@@ -25,16 +32,18 @@
 import org.openqa.selenium.firefox.FirefoxProfile;
 import org.openqa.selenium.logging.LogType;
 import org.openqa.selenium.logging.LoggingPreferences;
-import org.openqa.selenium.remote.CapabilityType;
-import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.remote.RemoteWebDriver;
 import org.openqa.selenium.safari.SafariDriver;
 import org.openqa.selenium.safari.SafariOptions;
-import java.io.File;
+import org.yaml.snakeyaml.Yaml;
+import java.io.*;
```

Comment:
are the total imports from java.io > 5, if not you can import specific classes.
same for java.util

Reply:
total imports from java.io  are > 4